pom.xml

```diff
@@ -637,11 +637,9 @@
             <reporting>
                 <plugins>
                     <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>findbugs-maven-plugin</artifactId>
+                        <groupId>com.github.spotbugs</groupId>
+                        <artifactId>spotbugs-maven-plugin</artifactId>
                         <configuration>
-                            <findbugsXmlOutput>true</findbugsXmlOutput>
-                            <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
```

Comment:
I assume these are no longer needed. Might be worth explaining why if others wonder in the future. :)

Reply:
It seems like spotbugs defaults are either different or these were superfluous anyway. Are you suggesting including a comment here? Seems a bit odd since once we're on spotbugs, nobody should care about findbugs anyway.

Reply:
I just meant as a comment in the PR as you have done.